### ISSUE TITLE
path normalization on windows

### ISSUE THREAD

**Issue by maiwenan** [jharris4/html-webpack-tags-plugin#44]
Hi, when using `html-webpack-tags-plugin` under windows, it generates the path link: `<link href=\lib\__MOEBIUS__.css?c69a3b2f8dc75215ce4a rel=stylesheet>`

![image](https://user-images.githubusercontent.com/4538563/56482407-99686980-64f6-11e9-8022-b050da682443.png)

here is my config:

```
{
  plugins: [
    new CopyWebpackPlugin([
      {
        from: 'dist-lib',
        to: 'lib/'
      }
    ]),
    new HtmlWebpackTagsPlugin({
      tags: [
        `lib/${PREFIX}.umd.min.js`,
        `lib/${PREFIX}.css`
      ],
      append: false,
      hash: true
    })
  ]
}
```

when I change the plugin code :

```
const DEFAULT_OPTIONS = {
  append: true,
  useHash: false,
  addHash: (assetPath, hash) => assetPath + '?' + hash,
  usePublicPath: true,
  addPublicPath: (assetPath, publicPath) => path.join(publicPath, assetPath),
  jsExtensions: ['.js'],
  cssExtensions: ['.css']
};
```

to 

```
const DEFAULT_OPTIONS = {
  append: true,
  useHash: false,
  addHash: (assetPath, hash) => assetPath + '?' + hash,
  usePublicPath: true,
  addPublicPath: (assetPath, publicPath) => slash(path.join(publicPath, assetPath)),
  jsExtensions: ['.js'],
  cssExtensions: ['.css']
};
```

it workds.

Any help would be much appreciated! Please let me know if I can provide any more details.

**Comment by jharris4** [comment]
Oh! I thought `path.join` was safe for windows! I'll go add that fix right now, thanks for reporting it!

**Comment by jharris4** [comment]
I just released version `2.0.4` with a fix, please confirm whether it fixes the issue for you.

**Comment by maiwenan** [comment]
`2.0.4` fixed the problem, thanks.
